components/clp-py-utils/clp_py_utils/initialize-privileged-db-user.py

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env python3
+import argparse
+import logging
+import pathlib
+import sys
+from contextlib import closing
+
+from pydantic import ValidationError
+from sql_adapter import SqlAdapter
+
+from clp_py_utils.clp_config import (
+    CLP_METADATA_TABLE_PREFIX,
+    ClpConfig,
+)
+from clp_py_utils.clp_metadata_db_utils import get_aws_credentials_table_name
+from clp_py_utils.core import read_yaml_config_file
+
+logger = logging.getLogger(__file__)
+logger.setLevel(logging.INFO)
+logging_console_handler = logging.StreamHandler()
+logging_formatter = logging.Formatter("%(asctime)s [%(levelname)s] %(message)s")
+logging_console_handler.setFormatter(logging_formatter)
+logger.addHandler(logging_console_handler)
+
+
+def main(argv):
+    args_parser = argparse.ArgumentParser(
+        description="Creates privileged database user for credential management."
+    )
+    args_parser.add_argument("--config", "-c", required=True, help="CLP configuration file.")
+    parsed_args = args_parser.parse_args(argv[1:])
+
+    config_path = pathlib.Path(parsed_args.config)
+    try:
+        clp_config = ClpConfig.model_validate(read_yaml_config_file(config_path))
+        clp_config.database.load_credentials_from_env()
```

Comment:
with the above refactoring proposal
```suggestion
        clp_config.database.load_credentials_from_env(ClpDbUserType.ROOT)
        clp_config.database.load_credentials_from_env(ClpDbUserType.ADMIN)
        clp_config.database.load_credentials_from_env(ClpDbUserType.CLP)
```